src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    if (obj.type === 'addProperties') {
+      Object.assign(state, obj.extraData);
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/213378315-46af3406-8dff-41e4-ae3d-a01b37c7bfe2.png)
Also, you can make a destructuring to get all the needed props of action(obj)